sechub-wrapper-prepare/src/main/java/com/mercedesbenz/sechub/wrapper/prepare/modules/git/GitPrepareWrapperModule.java

```diff
@@ -171,4 +173,11 @@ private void prepareRemoteConfiguration(GitContext gitContext, SecHubRemoteDataC
         gitWrapper.downloadRemoteData(gitContext);
     }
 
+    private String getRepositoryNameFromLocation(String location) {
```

Comment:
Hm...we have no unit tests for this!

I would prefer following here:

1. Inject `GitLocationConverter` via `@Autowire` to the git prepare module here.

2. Enhance existing `GitLocationConverter` class and introduce new method `convertToRepositoryName(String location)`

3. Use it here
4. Add some unit tests in `GitLocationConverterTest` for `convertToRepositoryName(String location)`

5. You could additionally add also a unit test for the wrapper, that the 
`GitLocationConverter.convertToRepositoryName(String location)` is the name for the output folder or at least that convert is called